fcrepo-webapp/src/main/webapp/WEB-INF/web.xml

```diff
@@ -34,6 +34,16 @@
 
 	</servlet-mapping>
 
+  <filter>
+    <filter-name>ETagFilter</filter-name>
+    <filter-class>org.springframework.web.filter.ShallowEtagHeaderFilter</filter-class>
```

Comment:
You would want to just send the filenames or something? Something that doesn't ever change and that we wouldn't need to keep up to date?